assets/app/view/map.rb

```diff
@@ -10,6 +10,7 @@ class Map < Snabberb::Component
     needs :game
     needs :tile_selector, default: nil, store: true
     needs :selected_company, default: nil, store: true
+    needs :opacity, default: nil
```

Comment:
i don't like this, but maybe it's a bigger change

while you're at it, instead of doing this

can you just fix this ? https://github.com/tobymao/18xx/issues/39

this should make the maps already non transparent and fix this issue

Reply:
Will take a crack at that tomorrow (probably late)

Reply:
e8c86c8 - still need to do privates